xray/commands/audit/nuget/nuget_test.go

```diff
@@ -0,0 +1,44 @@
+package nuget
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"testing"
+
+	"github.com/jfrog/build-info-go/entities"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/tests"
+
+	"github.com/jfrog/jfrog-client-go/xray/services"
+)
+
+func TestBuildNugetDependencyTree(t *testing.T) {
+	dependenciesJson, err := ioutil.ReadFile("../../testdata/nuget/dependencies.json")
+	if err != nil {
+		t.Error(err)
+	}
```

Comment:
Don't we prefer to use ```assertNoError``` instead of the above if? The same goes for other places in this file.